src/wasm-emscripten.cpp

```diff
@@ -86,6 +86,17 @@ std::vector<Function*> makeDynCallThunks(Module& wasm, std::vector<Name> const&
   return generatedFunctions;
 }
 
+void addGlobalImports(Module& wasm, std::unordered_set<cashew::IString> const& globalImports) {
+  for (Name name : globalImports) {
```

Comment:
Done.
Turns out `Name` didn't have an implementation of `hash` and `equal_to` whereas `cashew::IString` did, so I added those to `Name` so we could use it in sets.